### ISSUE TITLE
Add PassiveAggressiveRegressor

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#73]
Add PassiveAggressiveRegressor model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference PassiveAggressiveRegressor Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/PassiveAggressiveRegressor.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.PassiveAggressiveRegressor.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an PassiveAggressiveRegressor as a potential best fit model.

**Comment by melan96** [comment]
I will look this issue
